UpdatedImageProcessing/Classifiers/classify_tester.py

```diff
@@ -0,0 +1,156 @@
+import sys
+from os import walk
+import numpy as np
+import json
+import timeit
+
+from classify_color import ColorClassifier
+
+class ClassifierTester(object):
```

Comment:
This file should be located in ```tests/non_nose_tests/```

Reply:
This comment still applies; please move the test files to be located in the proper directory.